src/axom/inlet/Reader.hpp

```diff
@@ -198,19 +204,22 @@ class Reader
    */
   virtual bool getStringMap(const std::string& id,
                             std::unordered_map<int, std::string>& values) = 0;
+  virtual bool getStringMap(const std::string& id,
+                            std::unordered_map<std::string, std::string>& values) = 0;
 
   /*!
    *****************************************************************************
-   * \brief Get the list of indices for an array
+   * \brief Get the list of indices for an container
    *
-   * \param [in]  id    The identifier to the array that will be retrieved
-   * \param [out] map The values of the indices that were retrieved
+   * \param [in]  id    The identifier to the container that will be retrieved
+   * \param [out] indices The values of the indices that were retrieved
    *
-   * \return true if the array was able to be retrieved from the file
+   * \return true if the indices were able to be retrieved from the file
    *****************************************************************************
    */
-  virtual bool getArrayIndices(const std::string& id,
-                               std::vector<int>& indices) = 0;
+  virtual bool getIndices(const std::string& id, std::vector<int>& indices) = 0;
+  virtual bool getIndices(const std::string& id,
```

Comment:
Does doxygen automatically recognize this as an overload? and document both of them?

Reply:
I think I need to add a `/// \overload` before the overload - with that it generates the following:
![image](https://user-images.githubusercontent.com/68349992/100884922-ad68a100-3477-11eb-9bcb-ec6c54d9d081.png)